README.md

```diff
@@ -106,10 +127,25 @@ The FlowForge platform consists of a number of npm modules. Each module is maint
 in its own git repository. When developing the code and you need to make changes
 across multiple modules, you want to be sure the your development code is loaded.
 
-We used to achive that by running a script that modified the package.json of which
+We used to achieve that by running a script that modified the package.json of which
 module to point to a relative location. That left the package.json file modified
-and would get in the way when commiting and merging changes.
+and would get in the way when committing and merging changes.
 
 Thankfully, npm workspaces solves that particular problem. Whilst it is more
 often used in monorepos, this repo provides the setup required to get them working
 with our multiple repositories.
+
+
+## Other terminal tips
```

Comment:
These tips are not accurate.

With the npm workspace support, the proper way of doing this would be:

```
npm exec --ws -- <insert command here>
```

We already provides `npm run status` (as documented above) to get the status of each repo.

If there are other common tasks that need to be run against all repos at the same time, then we can add them.

I'm going to merge this as-is for the sake of needing to move forward - but this readme will need updating when there isn't much more higher priority work to get done in the next 24 hours.